content/concepts/core-concepts.md

```diff
@@ -2,12 +2,31 @@
 title: Core Concepts
 ---
 
-## Entry, Output, Loaders, Plugins
+## Overview
```

Comment:
@bebraw I'll take some input on this section. Since we are creating separate files. I felt like the 2-3 min overview is what this page is trying to accomplish. 

With that in mind, do you think the content speaks to that? Need more explanation, or less?